src/hotspot/share/opto/superword.cpp

```diff
@@ -69,7 +69,9 @@ SuperWord::SuperWord(PhaseIdealLoop* phase) :
   _nlist(arena(), 8, 0, NULL),            // scratch list of nodes
   _stk(arena(), 8, 0, NULL),              // scratch stack of nodes
   _lpt(NULL),                             // loop tree node
-  _lp(NULL),                              // LoopNode
+  _lp(NULL),                              // CountedLoopNode
+  _pre_loop_head(NULL),                   // Pre loop CountedLoopNode
```

Comment:
Is this really needed? Can't we just retrieve the pre-loop head via `_pre_loop_end->loopnode()`?

Reply:
You're right, it is not really needed. I removed it.